qtdialogs/DlgSetupManager.py

```diff
@@ -0,0 +1,1303 @@
+      widget.keyReleaseEvent   = MethodType(newKRE, widget)
+      widget.mousePressEvent   = MethodType(newMPE, widget)
+      widget.mouseReleaseEvent = MethodType(newMRE, widget)
+
+   def loadWalletList(self):
+      """Query bridge for wallets and render actions per type/state."""
+      self.walletList.clear()
+      self.walletIdToCheckbox.clear()
+      try:
+         addedAny = False
+         # Single bridge query; if it raises, show placeholder and return
+         try:
+            wltList = TheBridge.wltManager.listWallets()
+         except Exception as e:
+            addPlaceholderRow(
+               self.walletList, ['', self.tr('Bridge not ready'), '', ''])
+            return
+
+         # Pre-process to find migrated versions
+         migratedWalletIds = set()
+         for entry in wltList:
+            try:
+               walletId = entry.walletId
+               isStaged = entry.staged
+               if walletId and isStaged:
+                  migratedWalletIds.add(walletId)
+            except Exception:
+               continue
+
+         # Iterate directly; capnp lists are iterable
+         for entry in wltList:
+            if self._createWalletRow(entry, migratedWalletIds):
+               addedAny = True
+
+         # If still nothing, show guidance
+         if not addedAny:
+            addPlaceholderRow(self.walletList, [
+               self.tr(''),
+               self.tr('No wallets found.'),
+               self.tr(''),
+               self.tr('')
+            ])
+
+      except Exception:
+         # If wallet list loading fails completely, let it crash
+         # The UI will show the placeholder from the inner try/except
+         raise
+
+   def _addActionButton(self, item, text, handler):
+      """Create and add a left-aligned action button to the wallet list item."""
+      btn = QtWidgets.QPushButton(text)
+      btn.clicked.connect(handler)
+      btn.setMaximumWidth(60)  # Compact button
+      cellWidget = QtWidgets.QWidget()
+      layout = QtWidgets.QHBoxLayout(cellWidget)
+      layout.setContentsMargins(2, 2, 2, 2)  # Small margins
+      layout.addWidget(btn)
+      layout.addStretch()  # Push button to left
+      self.walletList.setItemWidget(item, 4, cellWidget)
+
+   def _createWalletRow(self, entry, migratedWalletIds=None):
+      """Create and add a wallet row from a bridge entry.
+
+      Returns True if the row is rendered successfully; False otherwise.
+      """
+      try:
+         walletPath = entry.path
+         fileName = os.path.basename(walletPath) if walletPath else ''
+         walletId = entry.walletId
+
+         # Check if this is a legacy wallet that has been migrated
+         # Per BridgeTests.cpp: after migration, there are 2 wallets with
+         # same walletId:
+         # - Legacy: staged=false (original file) <- Hide this one
+         # - Migrated: staged=true (new file) <- Show this one
+         try:
```

Comment:
sup with the try blocks everywhere? do you use those before taking a shit too?

`isStaged = entry.staged or False`